### DISCUSSION TITLE
PowerShell MSIX option

### DISCUSSION THREAD

**Discussion by VincentJoshuaET** [microsoft/winget-pkgs#275] [General] [answered]
I prefer to install the MSIX package. I hope it's added as well.

_Sent from my Redmi Note 7 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by TravisEz13** [comment, marked as answer]
We have a MSIX that will be published to the windows store.

Unfortunately, a standalone MSIX will have limited functionality (according to Appx this is by design) and we don't plan on releasing an MSIX.  

https://github.com/PowerShell/PowerShell/issues/7730

**Reply by roke-julian-lockhart** [reply]
> a standalone MSIX will have limited functionality (according to Appx this is by design)

@TravisEz13, what's the most modern installer format that provides you with the functionality you want, then? `.MSI`?

**Comment by roke-julian-lockhart** [comment]
https://github.com/PowerShell/PowerShell/discussions/14803#discussion-3223221
